atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/SweepStatsKeyValueService.java

```diff
@@ -107,24 +112,32 @@ public void put(TableReference tableRef, Map<Cell, byte[]> values, long timestam
         delegate().put(tableRef, values, timestamp);
         writesByTable.add(tableRef, values.size());
         recordModifications(values.size());
+        recordModificationsSize(values.entrySet().stream().mapToLong(cellEntry -> cellEntry.getValue().length)
```

Comment:
This adds linear overhead (the old tracking was constant), so benchmarks could be useful, as you mention. 

If we're concerned about performance, we could consider guarding the tracking behind a (live-reloadable) configuration flag.